server/app/com/xsn/explorer/models/request/TposContractsEncodeRequest.scala

```diff
@@ -0,0 +1,20 @@
+package com.xsn.explorer.models.request
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class TposContractsEncodeRequest(
+    tposAddress: String,
```

Comment:
Given that you are using a typed model, you can use the proper `Address` types`, for the signature, I believe we have a model to represent hex strings.